recipe/meta.yaml

```
@@ -10,26 +10,36 @@ source:
   sha256: 16148e8eba25fc364ed8feb00d3359f0a6249eb0a6e7c54c53598c73b87ba391
 
 build:
-  skip: true  # [win]
+  # For win, we don't have gettext, openblas 0.3.21.* and postgis
+  skip: true  # [win or s390x]
   number: 0
 
 requirements:
   build:
     - cmake
     - make
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
   host:
-    - proj
+    - proj {{ proj }}
     - json-c
     - gsl
     - postgis
-    - libcblas
+    - openblas {{ openblas }}
     - libpq
-    - geos
+    - geos {{ geos }}
+    - gdal
```

```suggestion
    - gdal  {{ libgdal }}
```
see https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L141

---

Done